test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -0,0 +1,859 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'google/apis'
+require 'grpc'
+require 'helper'
+require 'json'
+require 'mocha/test_unit'
+require 'time'
+require 'webmock/test_unit'
+
+require_relative 'base_test'
+
+# Unit tests for Google Cloud Logging plugin
+class GoogleCloudOutputGRPCTest < GoogleCloudPluginBaseTest
+  def test_configure_service_account_application_default_grpc
```

Comment:
I wonder if the `_grpc` suffixes are still useful here. Do Ruby tests get prefixed with the test class name? If yes, then the suffix is noise. If not, the suffix is needed.